.vscode/settings.json

```diff
@@ -9,5 +9,6 @@
   "flow.useNPMPackagedFlow": true,
   "javascript.validate.enable": false,
   "jest.pathToJest": "yarn jest --",
-  "prettier.eslintIntegration": true
+  "prettier.eslintIntegration": true,
+  "editor.formatOnSave": false
```

Comment:
is this needed?

Reply:
I prefer not

Reply:
i can revert the dedicated commit if you prefer that way.